rfpremier/ah-django
# #164857214 Receive confirmation email on successful registration

**What does this PR do?**
The PR lets a user receive a confirmation email once they successfully register

**Tasks to be completed**
- [x] Set up SMTP mail host, Sengrid in this instance.
- [x] Set up send email on successful registration
- [x] Write tests to confirm functionality

**How can this be manually tested?**
1. Clone this repo and check out the ft/164857214-email-verification

2. Create and activate a virtual environment

```
$ virtualenv env
$ source env/bin/activate
```
3. Install requirements
`$ pip install -r requirements.txt`

4. Make migrations and run the to update database
```
$ python manage.py makemigrations
$ python manage.py migrate
```
5. Start the application
`$ python manage.py runserver`

Open Postman and sign-up on http://127.0.0.1:8000/api/users/  using the following payload
```
{
          "user": {
                "email": "youremail@gmail.com",
                "username": "New Member",
                "password": "newmember000"
            }
        }
```
**NOTE:** 
substitute `youremail@gmail.com` with an email you have access to recieve and view the email.

If you successfully register, you should recieve an email in your inbox or spam folder.

**Pivotal Tracker story**
[#164857214](https://www.pivotaltracker.com/story/show/164857214)

**Screenshots**
![Screenshot (55)](https://user-images.githubusercontent.com/40719885/55156752-cc277800-516b-11e9-941f-d67b8d2107d9.png)
